hack/lib/common-var-init.sh

```diff
@@ -109,6 +109,7 @@ EXTERNAL_CLOUD_PROVIDER_BINARY=${EXTERNAL_CLOUD_PROVIDER_BINARY:-""}
 CLOUD_PROVIDER=${CLOUD_PROVIDER:-""}
 CLOUD_CONFIG=${CLOUD_CONFIG:-""}
 FEATURE_GATES=${FEATURE_GATES:-"AllAlpha=false"}
+FEATURE_GATES_COMMON_BASE=${FEATURE_GATES_COMMON_BASE:-"${FEATURE_GATES},WorkloadInfoDefaulting=true,QPSDoubleGCController=true,QPSDoubleRSController=true"}
```

Comment:
what is the reason a new env for FEATURE_GATES need be involved.

Reply:
After conversation with sonyafeng, the explanation is given.